drivers/lcd/st7789.c

```diff
@@ -80,16 +80,20 @@
       defined(CONFIG_LCD_RPORTRAIT)
 #    error "Cannot define both portrait and any other orientations"
 #  endif
+#  define LCD_STDORIENTATION LCD_PORTRAIT
 #elif defined(CONFIG_LCD_RPORTRAIT)
 #  if defined(CONFIG_LCD_LANDSCAPE) || defined(CONFIG_LCD_RLANDSCAPE)
 #    error "Cannot define both rportrait and any other orientations"
 #  endif
+#  define LCD_STDORIENTATION LCD_RPORTRAIT
 #elif defined(CONFIG_LCD_LANDSCAPE)
 #  ifdef CONFIG_LCD_RLANDSCAPE
 #    error "Cannot define both landscape and any other orientations"
 #  endif
+#  define LCD_STDORIENTATION LCD_LANDSCAPE
 #elif !defined(CONFIG_LCD_RLANDSCAPE)
 #  define CONFIG_LCD_LANDSCAPE 1
+#  define LCD_STDORIENTATION LCD_RLANDSCAPE
```

Comment:
If caller pass in orientation, why we keep LCD_STDORIENTATION?

Reply:
No

Reply:
not neccessary